server-spi-private/src/test/java/org/keycloak/validate/BuiltinValidatorsTest.java

```diff
@@ -462,5 +463,58 @@ public void validateUri() throws Exception {
 
         Assert.assertFalse(Validators.uriValidator().validateUri(new URI("http://customurl"), Collections.singleton("https"), true, true));
     }
+    
+    @Test
+    public void validateOptions(){
```

Comment:
Like other methods, could you please append the `test` prefix to all test methods?

Reply:
OK